### PR TITLE
Add ancient fakers

### PR DESCRIPTION
**Pull request by kkalavantavanich**
## Changes Introduced

Added fakers for "ancient" module. Also ordered entries in `default.conf`, `ja.conf`, and `en.conf` alphabetically.

## Applicable linked issues

#48 

## Checklist (check all that apply)

- [x] This change maintains backwards compatibility
- [x] I have introduced tests for all new features and changes
- [ ] I have added documentation covering all new features and changes
- [x] This pull-request is ready for review

### CODE REVIEW

**Comment by kkalavantavanich** [comment]
Checking...

File: src/main/resources/ja/ancient.conf [hunk 1]
```diff
@@ -0,0 +1,3 @@
+ancient {
+  gods: ["アフロディテ", "アポロン", "アレス", "アルテミス", "アテナ", "デメテル", "ディオニュソス", "ハデス", "ヘパイストス", "ヘラ", "ヘルメス", "ヘスティア", "ポセイドン", "ゼウス"]
```
Comment (etspaceman) [inline src/main/resources/ja/ancient.conf:2]: Let's be consistent with syntax and use `=` here
```suggestion
  gods = ["アフロディテ", "アポロン", "アレス", "アルテミス", "アテナ", "デメテル", "ディオニュソス", "ハデス", "ヘパイストス", "ヘラ", "ヘルメス", "ヘスティア", "ポセイドン", "ゼウス"]
```

File: src/test/scala/faker/FakerSpec.scala [hunk 2]
```diff
@@ -74,6 +74,12 @@ trait FakerSpec extends AnyFreeSpecLike with Checkers {
     testCanGen[address.StreetSuffix](locale, "address.StreetSuffix")(x =>
       Some(x.value)
     )
+    testCanGen[ancient.God](locale, "ancient.God")(x => Some(x.value))
```
Comment (etspaceman) [inline src/test/scala/faker/FakerSpec.scala:77]: Looks good on these tests. We need another set of tests for the `Faker` class as well. See the Currency section here: https://github.com/etspaceman/scalacheck-faker/blob/master/src/test/scala/faker/FakerSpec.scala#L888

**Review by etspaceman — CHANGES_REQUESTED** [review]
Thanks for the contribution! I added some code comments above. We also need these added to the `Faker` class here: https://github.com/etspaceman/scalacheck-faker/blob/master/src/main/scala/faker/Faker.scala

**Review by etspaceman — APPROVED** [review]
Thanks for the contribution!

**Pull request merged**